master/getting-started/rkt/installation/manual.md

```diff
@@ -5,43 +5,40 @@ title:  Manual installation of Calico with rkt
-named `calico-01` and `calico-02`.  These names are not required to run the tutorials
-but you may need to adapt the tutorial instructions based on your actual node names.
-
 * TOC
 {:toc}
 
 ## System requirements
 
 - A cluster of bare metal servers or VMs (nodes) with a modern 64-bit Linux OS and IP connectivity
   between them.
-- A recent version of [`rkt`][https://github.com/coreos/rkt/releases/latest] installed on each node in the cluster.  We recommend
+- A recent version of [`rkt`](https://github.com/coreos/rkt/releases/latest) installed on each node in the cluster.  We recommend
   version > 1.20.0 as this is contains important fixes that prevent leaking IP addresses
   when containers are deleted.
-- An [`etcd`][https://coreos.com/etcd/docs/latest/] cluster accessible by all nodes in the cluster
+- An [`etcd`](https://coreos.com/etcd/docs/latest/) cluster accessible by all nodes in the cluster
 
 ## About the Calico Components
 
 There are three components of a Calico / rkt integration.
 
-- The Calico per-node rkt container, [calico/node](https://quay.io/repository/calico/node?tab=tags)
-- The [`calicoctl`](https://github.com/projectcalico/calico-containers) binary used to manage network
-  policy.
-- The [calico-cni](https://github.com/projectcalico/calico-cni) network plugin binaries.
-  - This is the combination of two binary executables and a configuration file.
+#### 1. The calico/node container
+
+The Calico per-node rkt container, [calico/node](https://quay.io/repository/calico/node?tab=tags),
+must be run on every node in your cluster.  It contains the BGP agent which provides Calico routing,
+and the Felix agent which programs network policy rules.
+
+#### 2. The calicoctl CLI tool
 
-The `calico/node` rkt container must be run on node in your cluster.  It contains 
-the BGP agent necessary for Calico routing to occur, and the Felix agent which programs 
-network policy rules.
+The [`calicoctl`](https://github.com/projectcalico/calico-containers) binary is
+a command line utility that can be used to manage network policy for your rkt 
+containers, and can be used to monitor the status of your Calico services.
 
-The `calicoctl` binary is a command line utility that can be used to manage network
-policy for your rkt containers, and can be used to monitor the status of your
-Calico services.
+#### 3. The calico-cni plugin binaries
 
-The `calico-cni` plugin binaries integrate directly with the rkt container lifecycle 
-hooks on each node to configure the container interfaces, manage IP addresses and 
-configure the containers to use Calico policy.
+The [calico-cni](https://github.com/projectcalico/calico-cni) network plugin binaries
+are a combination of two binary executables.  These binaries are invoked from  
+the rkt container lifecycle hooks on each node to configure the container interfaces, 
+manage IP addresses and enable Calico policy on the containers.
 
 ## Installing `calico/node` 
```

Comment:
You switch between "Installing" and "Install" a lot in these headings. I think I prefer "Install".